dnpchallenge/iedconfig.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <ot-sim>
   <message-bus>
-    <pull-endpoint>tcp://127.0.0.1:1234</pull-endpoint>
-    <pub-endpoint>tcp://127.0.0.1:5678</pub-endpoint>
+    <pull-endpoint>tcp://127.0.0.1:9012</pull-endpoint>
+    <pub-endpoint>tcp://127.0.0.1:3456</pub-endpoint>
```

Comment:
When running two ot-sim devices on the same host, they each need to be configured with unique message bus endpoints. Otherwise, there will be message bus cross-talk between the devices.